app/tests/spec/views/settings.js

```diff
@@ -30,7 +28,7 @@ function (chai, $, sinon, View, RouterMock, WindowMock, TestHelpers,
   describe('views/settings', function () {
     var view;
     var routerMock;
-    var windowMock;
+    var windowMock; //eslint-disable-line no-unused-vars
```

Comment:
`windowMock` isn't used in this file, so let's remove it and the `WindowMock` module so that amdcheck passes.